src/libraries/Microsoft.Win32.Registry/README.md

```diff
@@ -0,0 +1,15 @@
+# Microsoft.Win32.Registry
+
+This assembly provides APIs for accessing and modifying the Windows Registry.
+
+## Contribution Bar
+
+- [x] [We only consider fixes to maintain or improve quality](../README.md#primary-bar)
+- [x] [We consider PRs that target this library for new source code analyzers](../README.md#secondary-bars)
+- [x] [We consider PRs with refactoring changes due to new language features](../README.md#secondary-bars)
+
+See the [Help Wanted](https://github.com/dotnet/runtime/issues?q=is%3Aopen+is%3Aissue+label%3Aarea-Microsoft.Win32) issues.
+
+## Deployment
+
+`Microsoft.Win32.Registry` is provided as a [NuGet package](https://www.nuget.org/packages/Microsoft.Win32.Registry/).
```

Comment:
Also incomplete: https://github.com/dotnet/runtime/blob/493574ee470daf8a353203ce7e933f931f986159/src/libraries/NetCoreAppLibrary.props#L34

Needs to be like https://github.com/dotnet/runtime/pull/78121/files#r1018292210